docs/docs/administration/postgres-standalone.md

```diff
@@ -0,0 +1,45 @@
+# Preparing up a standalone Postgres server
```

Comment:
```suggestion
# Preparing a pre-existing Postgres server
```
I think "pre-existing" is more descriptive here. It's unclear what "standalone" means without more context.